examples/tutorials/self-paced-training/part-3_security_and_privacy/chapter-5_Privacy_In_Federated_Learning/05.3_homomorphic_encryption/05.3.1_privacy_with_homormorphic_encryption.ipynb

```diff
@@ -0,0 +1,284 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "5c0688f9-ae01-48e9-9411-3b1143c98962",
+   "metadata": {},
+   "source": [
+    "# Federated Learning with Homomorphic Encryption for Secure Aggregation\n",
+    "\n",
+    "Please make sure you set up a virtual environment and follow [example root readme](../../README.md) before starting this notebook.\n",
+    "Then, install the requirements.\n",
+    "\n",
+    "<div class=\"alert alert-block alert-info\"> <b>NOTE</b> Some of the cells below generate long text output.  We're using <pre>%%capture --no-display --no-stderr cell_output</pre> to suppress this output.  Comment or delete this line in the cells below to restore full output.</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3f877106-9455-45ed-ad81-6f556f24303b",
+   "metadata": {
+    "jupyter": {
+     "source_hidden": true
+    }
+   },
+   "outputs": [],
+   "source": [
+    "%%capture --no-display --no-stderr cell_output\n",
+    "import sys\n",
+    "!{sys.executable} -m pip install -r requirements.txt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fdf0d7a9-a693-447d-9390-7cdec51a5936",
+   "metadata": {},
+   "source": [
+    "### Homomorphic Encryption in NVFlare\n",
+    "This example includes instructions on running [FedAvg](https://arxiv.org/abs/1602.05629) with [homomorphic encryption](https://developer.nvidia.com/blog/federated-learning-with-homomorphic-encryption/) (originally developed for Clara Train's FL feature, the predecessor of NVFlare).\n",
```

Comment:
Do we need to mention "originally developed for Clara Train's FL feature, the predecessor of NVFlare"?

Reply:
I'm mentioning it because the linked blog post mentions Clara Train rather than NVFlare.